model/project_ref.go

```diff
@@ -854,7 +851,6 @@ func (p *ProjectRef) createNewRepoRef(u *user.DBUser) (repoRef *RepoRef, err err
 	}
 	// some fields shouldn't be set from projects
 	repoRef.Id = mgobson.NewObjectId().Hex()
-	repoRef.UseRepoSettings = false
```

Comment:
I'm not sure why we don't already explicitly make sure RepoRefId is empty but it should be.

Reply:
I added an error if it's not empty. Is that what you meant?

Reply:
Sorry, I meant that here the line that was deleted: `repoRef.UseRepoSettings = false` should probably be replaced with `repoRef.RepoRefId = ""` (since that's not a real field for repos).